lighthouse-core/report/v2/renderer/dom.js

```diff
@@ -112,6 +123,16 @@ class DOM {
     }
     return result;
   }
+
+  /**
+   * Helper for context.querySelectorAll. Returns an Array instead of a NodeList.
+   * @param {string} query
+   * @param {!DocumentFragment|!Element} context
+   * @return {!Array<Element>}
```

Comment:
should be `{!Array<!Element>}`. Technically fine here, since `Element` is part of `null|Element`, but closure should be complaining where this is being called and null isn't checked.

Looks like the typing is being lost in the arrow functions passed to `forEach`, but the old type inference isn't complaining and the NTI has too many bugs elsewhere to turn on yet :(